ClientApp/package.json

```diff
@@ -8,32 +8,43 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
+    "@mdi/font": "^5.8.55",
+    "axios": "^0.21.1",
     "core-js": "^3.6.5",
     "date-fns": "^2.16.1",
     "register-service-worker": "^1.7.1",
     "vue": "^2.6.12",
     "vue-class-component": "^7.2.6",
-    "vue-property-decorator": "^9.0.2",
-    "vue-router": "^3.4.8",
-    "vuetify": "^2.3.15",
-    "vuex": "^3.5.1",
+    "vue-property-decorator": "^8.4.2",
+    "vue-router": "^3.4.9",
+    "vuetify": "^2.4.0",
+    "vuex": "^3.6.0",
     "vuex-class": "^0.3.2"
   },
   "devDependencies": {
-    "@mdi/font": "^5.8.55",
-    "@types/node": "^14.14.5",
-    "@vue/cli-plugin-babel": "^4.5.8",
-    "@vue/cli-plugin-pwa": "^4.5.8",
-    "@vue/cli-plugin-typescript": "^4.5.8",
-    "@vue/cli-service": "^4.5.8",
-    "axios": "^0.21.0",
-    "babel-plugin-transform-imports": "^2.0.0",
-    "sass": "^1.27.0",
-    "sass-loader": "^10.0.4",
-    "typescript": "^4.0.5",
-    "vue-cli-plugin-axios": "0.0.4",
-    "vue-cli-plugin-vuetify": "^2.0.7",
-    "vue-template-compiler": "^2.6.12",
-    "vuetify-loader": "^1.6.0"
+    "@typescript-eslint/eslint-plugin": "^2.33.0",
+    "@typescript-eslint/parser": "^2.33.0",
+    "@vue/cli-plugin-babel": "~4.5.0",
+    "@vue/cli-plugin-eslint": "~4.5.0",
+    "@vue/cli-plugin-pwa": "~4.5.0",
+    "@vue/cli-plugin-router": "~4.5.0",
+    "@vue/cli-plugin-typescript": "~4.5.0",
+    "@vue/cli-plugin-vuex": "~4.5.0",
+    "@vue/cli-service": "~4.5.0",
+    "@vue/eslint-config-standard": "^5.1.2",
+    "@vue/eslint-config-typescript": "^5.0.2",
+    "eslint": "^6.7.2",
+    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^4.2.1",
+    "eslint-plugin-standard": "^4.0.0",
+    "eslint-plugin-vue": "^6.2.2",
+    "node-sass": "^4.12.0",
+    "sass": "^1.19.0",
```

Comment:
Why are some packages downgraded? Is there a known issue with packages like `sass` or `vue-property-decorator` when updated to the latest versions?

Reply:
No, I just kept the ones that vuecli put in.  I know we can't go to ESLint 7 because there are a few dependencies not yet updates for it.  I will review the others and be sure we aren't downgrading from earlier versions of the template.

Reply:
I guess `sass` shouldn't be a problem to upgrade again to the latest 1.32.0.
More interesting is `typescript` and `vue-property-decorator`, that were downgraded 4 -> 3 and 9 -> 8

Reply:
Fixed those.   ESLint can't go to 7, due to vue/cli-plugin-eslint which is dependent on 6.x.  Probably have to wait for vue-cli v5 for that.